### ISSUE TITLE
Can we make long polling?

### ISSUE THREAD

**Issue by roysG** [vshymanskyy/TinyGSM#525]
I need to hold the connection open of 45 sec, can i doing it with this library?

**Comment by SRGDamia1** [comment]
Yes, you can have a connection open for 45s or longer.  The default connection timeout depends on the module, but I think they're all capable of long or indefinite connections.

**Comment by roysG** [comment]
Good news, can i get example or show direct me to manual how to do it?

**Comment by SRGDamia1** [comment]
Have you tried just opening a connection and not closing it?

What module are you using?

**Comment by gorghino** [comment]
@roysG Did you get a infinite connection?
